app/assets/javascripts/drawer.js

```diff
@@ -0,0 +1,11 @@
+function openDrawer() {
+  let page = document.getElementById("page_container");
+
+  page.classList.add("active");
+}
+
+function closeDrawer() {
+  let page = document.getElementById("page_container");
+
+  page.classList.remove("active");
```

Comment:
I really don't think we can use the scrunching method for mobile. At some point, the content needs to slide out of the way (or be covered). Otherwise we get this:
<img width="473" alt="screen shot 2018-09-20 at 3 15 43 pm" src="https://user-images.githubusercontent.com/6565554/45841545-268cdb00-bce8-11e8-9386-ae0a3259aa8f.png">